### PR TITLE
fix documentation issues

### PR DESCRIPTION
**Pull request by psychocrypt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,12 +28,12 @@ About 10% (~8.8 million) was premined to reserve for future development, i.e. **
...
 
-- **Coin symbol**: **SUMO**
+- **Coin symbol**: **RYO**
 - **Coin Units**:
-  + 1 Sumoshi &nbsp;= 0.000000001 **SUMO** (10<sup>-9</sup> - _the smallest coin unit_)
-  + 1 Sumokun = 0.000001 **SUMO** (10<sup>-6</sup>)
-  + 1 Sumosan = 0.001 **SUMO** (10<sup>-3</sup>)
-- **Hash algorithm**: CryptoNight (Proof-Of-Work)
+  + 1 Ryo &nbsp;= 0.000000001 **RYO** (10<sup>-9</sup> - _the smallest coin unit_)
+  + 1 Ryo = 0.000001 **RYP** (10<sup>-6</sup>)
```
Comment (Technohacker) [inline README.md]: Potentially confusing. 1 Ryo = 0.001 Ryo = 1 Ryo?

File: README.md [hunk 2]
```diff
@@ -28,12 +28,12 @@ About 10% (~8.8 million) was premined to reserve for future development, i.e. **
...
+  + 1 Ryo = 0.001 **RYO** (10<sup>-3</sup>)
+- **Hash algorithm**: cryptonight_heavy (Proof-Of-Work)
```
Comment (Technohacker) [inline README.md]: I'd suggest "CryptoNight Heavy"

File: README.md [hunk 3]
```diff
@@ -28,12 +28,12 @@ About 10% (~8.8 million) was premined to reserve for future development, i.e. **
...
 The pre-mined coins have been frozen/burned in commit [c3a3cb6](https://github.com/ryo-currency/ryo-emergency/commit/c3a3cb620488e88be7c52e017072261a3063b872)/ [blockchain_db/blockchain_db.cpp#L250-L258](https://github.com/ryo-currency/ryo-emergency/blob/c3a3cb620488e88be7c52e017072261a3063b872/src/blockchain_db/blockchain_db.cpp#L250-L258) as announced on [reddit](https://www.reddit.com/r/ryocurrency/comments/8nb8eq/direction_for_ryo/).
 
-- **Coin symbol**: **SUMO**
+- **Coin symbol**: **RYO**
 - **Coin Units**:
-  + 1 Sumoshi &nbsp;= 0.000000001 **SUMO** (10<sup>-9</sup> - _the smallest coin unit_)
-  + 1 Sumokun = 0.000001 **SUMO** (10<sup>-6</sup>)
-  + 1 Sumosan = 0.001 **SUMO** (10<sup>-3</sup>)
-- **Hash algorithm**: CryptoNight (Proof-Of-Work)
+  + 1 nanoRyo &nbsp;= 0.000000001 **RYO** (10<sup>-9</sup> - _the smallest coin unit_)
```
Comment (Technohacker) [inline README.md]: Minor: Unnecessary `&nbsp;`

File: README.md (hunk 2 above)
Comment (psychocrypt) [reply README.md]: ok wil change it

File: README.md (hunk 1 above)
Comment (psychocrypt) [reply README.md]: no this will be confusing but this should be well known https://en.wikipedia.org/wiki/Unit_prefix

File: README.md (hunk 3 above)
Comment (psychocrypt) [reply README.md]: fixed

**Pull request merged**